File: Vol.11(2000-2099)/#2022 Factorial/p2022.cpp
#include <cstdio>
int main() {
	int	N;
	scanf( "%d", &N );
	while( N-- ) {
		int	temp,	ans = 0;
		scanf( "%d", &temp );
		for( int p = 5; p <= temp; p *= 5 )
			ans += temp / p; 
		printf( "%d\n", ans ); 
	} 
} 

File: 范例/Accpt/20XX/p2022.pas
PROGRAM p2022;

VAR
        n,i                             :Integer;
        ans                             :Longint;
        temp                            :Longint;

BEGIN
readln(n);
for i:=1 to n do
        begin
        readln(temp);
        ans:=0;
        while temp>0 do
                begin
                inc(ans,temp div 5);
                temp:=temp div 5;
                end;
        writeln(ans);
        end;
END.